example/android/gradle.properties

```diff
@@ -2,4 +2,4 @@ org.gradle.jvmargs=-Xmx1536M
 org.gradle.configureondemand=false
 android.useAndroidX=true
 android.enableJetifier=true
-android.enableR8=true
+android.enableD8=true
```

Comment:
enableR8 deprecated according to AS.
Followed recommendation here https://stackoverflow.com/a/60594296/1240193 to add D8 instead.

Reply:
AFAICT, this is not needed at all, and we should remove this property. D8 the default dexer for quite some time now (it's been the default after `dx` since about 2018 IIRC). Also R8 has replaced Proguard and is not the default minifier/obfucsator for Android apps. Both tools are working together in a single step (with D8 being the frontend tool). By simply removing this setting, the default should still be D8/R8, which is what we want.